templates/config/initializers/devise.rb

```diff
@@ -0,0 +1,3 @@
+Devise.setup do |config|
+  config.mailer = 'UserMailer'
+end
```

Comment:
I added a new initializer to override the Devise mailer and set it to the local UserMailer.